cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java

```diff
@@ -404,6 +404,12 @@ private File createModuleJarFile(Class<?> cls) throws IOException {
   }
 
   @Override
+  public void deleteDatasetInstance(NamespaceId namespaceId, String datasetInstanceName) throws Exception {
```

Comment:
No this is not required, its not used anywhere. I think I accidently committed it. Removing it.